cli/src/test/java/com/devonfw/tools/ide/tool/MvnTest.java

```diff
@@ -0,0 +1,30 @@
+package com.devonfw.tools.ide.tool;
+
+import java.nio.file.Files;
+import java.nio.file.Path;
+
+import org.junit.jupiter.api.Test;
+
+import com.devonfw.tools.ide.context.AbstractIdeContextTest;
+import com.devonfw.tools.ide.context.IdeTestContext;
+import com.devonfw.tools.ide.tool.mvn.Mvn;
+
+/**
+ * Test of {@link Mvn}.
+ */
+public class MvnTest extends AbstractIdeContextTest {
+
+  @Test
+  public void testMvnPostInstall() {
+
+    // arrange
+    IdeTestContext context = newContext(PROJECT_BASIC, "", false);
```

Comment:
```suggestion
    IdeTestContext context = newContext(PROJECT_BASIC, "", true);
```

Reply:
With this option set to `false` the contents of `cli/src` are changed. Don't we want to avoid this?